pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -412,6 +413,75 @@ func TestPriorityQueue_Delete(t *testing.T) {
 	}
 }
 
+func TestPriorityQueue_Activate(t *testing.T) {
+	var objs []runtime.Object
+	q := NewTestQueueWithObjects(context.Background(), newDefaultQueueSort(), objs)
```

Comment:
If the tests are supposed to be independent of each other, create a new queue for each.